### PR TITLE
feat(experimental): add `signatureNotifications` websocket method

### PR DESCRIPTION
**Pull request by buffalojoec**

### CODE REVIEW

**Comment by buffalojoec** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1643** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1643" target="_blank">Graphite</a> 
    * **PR #1644** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1644" target="_blank">Graphite</a> 
      * **PR #1645** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1645" target="_blank">Graphite</a> 
        * **PR #1646** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1646" target="_blank">Graphite</a> 
          * **PR #1647** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1647" target="_blank">Graphite</a> 
            * **PR #1648** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1648" target="_blank">Graphite</a> 
              * **PR #1649** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1649" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1649?utm_source=stack-comment).

**Comment by buffalojoec** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1643** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1643" target="_blank">Graphite</a> 
    * **PR #1644** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1644" target="_blank">Graphite</a> 
      * **PR #1645** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1645" target="_blank">Graphite</a> 
        * **PR #1646** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1646" target="_blank">Graphite</a> 
          * **PR #1647** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1647" target="_blank">Graphite</a> 
            * **PR #1648** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1648" target="_blank">Graphite</a> 
              * **PR #1649** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1649" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1649?utm_source=stack-comment).

**Review by steveluscher — APPROVED** [review]
[closepuppy1](https://app.graphite.dev/macros/solana-labs)

**Review by steveluscher — CHANGES_REQUESTED** [review]
.

File: packages/rpc-core/src/rpc-subscriptions/signature-notifications.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+type SignatureNotificationsApiNotification = RpcResponse<
+    Readonly<{
+        // Error if transaction failed, null if transaction succeeded.
+        err: TransactionError | null;
+    }>
+>;
+
+type SignatureNotificationsApiConfig = Readonly<{
+    commitment?: Commitment;
```
[the remark this answers is not in the document]
Comment (buffalojoec) [reply packages/rpc-core/src/rpc-subscriptions/signature-notifications.ts:13]: Ahh nuts, must have hustled through this one. Adding!

[the remark this answers is not in the document]
Comment (buffalojoec) [reply packages/rpc-core/src/rpc-subscriptions/signature-notifications.ts:13]: Added!

**Pull request merged**